### DISCUSSION TITLE
[Question] Usage in local machine

### DISCUSSION THREAD

**Discussion by excellencemichel** [serverpod/serverpod#377] [Q&A] [answered]
I will be possible to deploy in local machine, maybe when we are making CRM that can work in local mode ?

**Comment by vlidholt** [comment, marked as answer]
Yes, you can run your Serverpod anywhere. You just need a server that can run Dart and has access to a Postgres database.

**Comment by excellencemichel** [comment]
Okay nice, because I develop in Odoo instance which can do this (PostgrSQL Server db and App code in Python) I want to move my Odoo app to Dart/Flutter
